### PR TITLE
add note that pinafore is unmaintained and fix mammoth link

### PR DESCRIPTION
**Pull request by CyberFlameGO**

### CODE REVIEW

**Comment by CyberFlameGO** [comment]
just one other sidenote (which should probably be a github issue but idk) - it seems like elk.zone doesn't actually work? i'm not sure if anyone else CR this but just a sidenote

File: docs/supported-clients.md [hunk 1]
```diff
@@ -35,8 +36,9 @@ Wildebeest also provides a read-only web client in your instance URL, where you
 [9]: https://github.com/elk-zone/elk
 [10]: https://tooot.app/
 [11]: https://github.com/tooot-app
-[12]: https://testflight.apple.com/join/66c1wW8y
+[12]: https://testflight.apple.com/join/ejJ70WEq
```
Comment (xtuc) [inline docs/supported-clients.md:39]: why does this link change? can't we use a stable link?

File: docs/supported-clients.md [hunk 2]
```diff
@@ -8,17 +8,18 @@ This is the list clients that we have been using successfully while developing a
...
-| [Mammoth][10]         | Mobile  | yes   | yes  | yes    | yes    | yes    | n/a             | [github][11] | ✅ <sup>2</sup> <sup>3</sup>
+| [Ivory for iOS][7]    | Mobile  | yes   | ?    | ?      | ?      | ?      | ?               | n/a          | ❌ <sup>2</sup>
+| [Elk][8]              | Web     | yes   | yes  | yes    | yes    | yes    | yes             | [github][9]  | ✅ <sup>3</sup>
+| [Tooot][10]           | Mobile  | yes   | yes  | yes    | yes    | yes    | n/a             | [github][11] | ✅ <sup>3</sup>
+| [Mammoth][12]         | Mobile  | yes   | yes  | yes    | yes    | yes    | n/a             | n/a          | ✅ <sup>3</sup> <sup>4</sup>
 
-<sup>1</sup> we know what the issue is and are in contact with the author<br/>
-<sup>2</sup> [conversations][conversations], [favourites][favorites], [bookmarks][bookmarks] and [trends][trends] are not implemented yet<br/>
-<sup>3</sup> notification filters are broken
+<sup>1</sup> [unmaintained][pinafore-unmaintained]<br/>
```
Comment (xtuc) [inline docs/supported-clients.md:19]: this sounds like Wildebeest isn't maintaining compatiblity with pinafore. Can we rephrase that to `client unmaintained`?

File: docs/supported-clients.md (hunk 1 above)
Comment (CyberFlameGO) [reply docs/supported-clients.md:39]: I'm not too sure - most of the time TestFlight links are stable  but the developer might've recreated the app on Apple's side (I only found the new one by looking through the developer's recent Mastodon toots where they said the old link's build won't be updated)

File: docs/supported-clients.md (hunk 2 above)
Comment (CyberFlameGO) [reply docs/supported-clients.md:19]: definitely! I'll get right on it

File: docs/supported-clients.md (hunk 1 above)
Comment (xtuc) [reply docs/supported-clients.md:39]: It says `This beta isn't accepting any new testers right now.`, the previous link says `To join the Mammoth: for Mastodon beta, open the link on your iPhone, iPad, or Mac after you install TestFlight.`

can you please revert that change? maybe we can ask the authors?

Comment (CyberFlameGO) [reply docs/supported-clients.md:39]: Alright I'll revert it now!

**Review by xtuc — APPROVED** [review]

**Pull request merged**